R/utilities-sensitivity-analysis.R

```diff
@@ -403,30 +386,78 @@ runPopulationSensitivityAnalysis <- function(structureSet, settings, logFolder =
 }
 
 
+#' @title getPKResultsDataFrame
+#' @description Read PK parameter results into a dataframe and set QuantityPath,Parameter and Unit columns as factors
+#' @param structureSet `SimulationStructure` R6 class object
+#' @return pkResultsDataFrame, a dataframe storing the contents of the CSV file with path pkParameterResultsFilePath
+#' @import ospsuite
+getPKResultsDataFrame <- function(structureSet) {
+  pkResultsDataFrame <- read.csv(structureSet$pkAnalysisResultsFileNames,
+    encoding = "UTF-8",
+    check.names = FALSE,
+    stringsAsFactors = FALSE
+  )

```

Comment:
Why not using opsuite::importPKAnalysesFromCSV + opsuite::pkAnalysesAsDataFrame intead ?

Reply:
Fixed